### ISSUE TITLE
As an applicant making updates to my "on hold" NCGU application, I don't want to have errors in unexpected fields (Email) so that I can submit easily. 

### ISSUE THREAD

**Issue by ASprinkle** [USDAForestService/fs-open-forest-platform#1222]
## Acceptance Criteria
- [ ]

## Tasks
- [ ] 
- [ ] PO approved

## Definition of Done
- [ ] Pull requests meet technical definition of done
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by ASprinkle** [comment]
@briandavidson this is a tag along to 1194 - it looks like 1194 was only completed for the TOG application. So I created this story to also updated the NCGU edit application instance.

**Comment by briandavidson** [comment]
Hmm, I thought the component I updated for 1194 was a shared component that got used in both places. Clearly not! -- I'll patch this up, sorry about that @ASprinkle !

**Comment by briandavidson** [comment]
> Hmm, I thought the component I updated for 1194 was a shared component that got used in both places. Clearly not! -- I'll patch this up, sorry about that @ASprinkle !

^^ strike that -- that is wrong. this was most definitely not a shared component. patching it up now.

**Comment by ASprinkle** [comment]
@briandavidson When moving a story from in progress to awaiting acceptance can you please also check the task and DoD boxes? This way I know that stuff is all good.

**Comment by briandavidson** [comment]
I'll step it up with my box ticking, my fault @ASprinkle !

**Comment by mtlaney** [comment]
blocked by #1196
